File: Backend/Backend_ProyectoFinal/Backend_ProyectoFinal/Models/EmpleadoTI.cs
namespace Backend_ProyectoFinal.Models
{
    public class EmpleadoTI : Usuario
    {
        public EmpleadoTI(string user, string contra, string nombre, string apellido, string correo) 
            : base(user, contra, nombre, apellido, correo)
        {
            this.TipoUsuario = "EmpleadoTI";
        }

        public EmpleadoTI()
        {

        }

        public bool RegistrarActividad(Actividad a)
        {
            return false;
        }
    }
}

File: main/java/com/example/appmovil/model/EmpleadoTI.java
package com.example.appmovil.model;

public class EmpleadoTI extends Usuario{

    public EmpleadoTI(String user, String contra, String nombre, String apellido, String correo) {
        super(user, contra, nombre, apellido, correo);
        this.setTipoUsuario("EmpleadoTI");
    }


    public EmpleadoTI()
    {

    }

    public boolean RegistrarActividad(Actividad a)
    {
        return false;
    }
}
